src/Sparrow/Json/AbstractBlittableJsonTextWriter.cs

```diff
@@ -205,12 +214,17 @@ public int WriteDateTime(DateTime? value, bool isUtc)
             return WriteDateTime(value.Value, isUtc);
         }
 
+#if NET6_0_OR_GREATER
+        [SkipLocalsInit]
+#endif
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public int WriteDateTime(DateTime value, bool isUtc)
         {
-            int size = value.GetDefaultRavenFormat(_auxiliarBuffer, _auxiliarBufferLength, isUtc);
+            var auxBuffer = stackalloc byte[32];
```

Comment:
Why is it more efficient to do the allocation every time instead of using `_auxiliarBuffer`?

Reply:
Because of cache behavior. The stack is already hot.

Reply:
There is also a sizable win because we don't have to allocate the auxiliary buffer when we create the writer. There is not a higher performance code, than code you dont need to execute :D